src/components/Button/Button.story.jsx

```diff
@@ -23,6 +25,7 @@ stories.add(
           color={select('Color', ['default', 'primary', 'secondary'], 'default')}
           disabled={boolean('Disabled', false)}
           type="button"
+          icon={select('icon', Object.keys(iconNames), 'edit')}
```

Comment:
make the default icon for the story `null` or maybe `none` so that it doesn't look like the icon is required for the button component